.github/workflows/ci-skip.yml

```diff
@@ -1,7 +1,15 @@
 name: CI skip
 
 on:
-  pull_request:
+  # We use `pull_request_target` to have access to the necessary secrets to
+  # create the commit status in PRs created from forks. We avoid command
+  # injection by:
+  # - never using the PR's branch name.
+  # - passing the PR's repository name through an intermediate environment
+  #   variable.
+  # - never running commands with the fork's code checked out (we only require
+  #   its commits to be present in the repository).
+  pull_request_target:
```

Comment:
This workflow won't run for this PR because of this change:
- The workflow in this branch uses `pull_request_target`, so won't run in the context of the PR.
- The workflow currently in `master` uses `pull_request`, so won't run in the context of the base branch.

Once we've admin-merged this PR, subsequent PRs should trigger fine, but it will continue to be the case that changes to `ci-skip.yml` won't actually start being used until after the PR they are in has merged.